### ISSUE TITLE
Clarification on high-resolution finetuning

### ISSUE THREAD

**Issue by goutamyg** [valeoai/Franca#9]
Hi,

1. In the [high-resolution finetuning cfg file](https://github.com/valeoai/Franca/blob/main/franca/configs/finetune/vitl14_finetune.yaml), the student network is initialized from teacher ckpt from the previous pre-training stage. What about the teacher network? Is it trained from scratch in the finetuing stage?

2. What will be the range of values of dino and ibot loss functions during finetuning? Will it be same as the pre-training? I am suspecting it shouldn't, since the student network is initialized with pretrained weights. 

3. Post the finetuning, do you use resulting teacher or the student network for downstream tasks? Since the finetuning duration is extremely small (compared the pre-training), I am wondering how well-trained is the teacher network for dense prediction tasks. 

Thank you in advance!

-- Goutam

**Comment by shashankvkt** [comment]
Hi @goutamyg 
Sorry for the late reply
1. The params of the teacher is updated using EMA of the student. The teacher is never trained.
2. It starts the same as pretraining, but you should see a very quick convergence.. Like if you set OFFICIAL_EPOCH_LENGTH=100, you should see the loss converge in 1250 iterations or so
3. We always use the weights from the teacher for downstream task.

**Comment by shashankvkt** [comment]
@goutamyg 
closing this issue for now, but feel free to reopen if you have any follow-up questions.
